### PR TITLE
Fixes V3-173 - ECClient check if given nodes are unique

### PR DESCRIPTION
**Pull request by kaloyan-raev**
Fixes https://storjlabs.atlassian.net/browse/V3-173

---
This change is [Reviewable](https://reviewable.io/reviews/storj/storj/186)

### CODE REVIEW

File: pkg/storage/ec/client.go [hunk 1]
```diff
@@ -203,3 +207,24 @@ func closeConn(ps client.PSClient, nodeID string) {
...
+		return true
+	}
+
+	ids := make([]string, len(nodes))
+	for i, n := range nodes {
+		ids[i] = n.GetId()
+	}
+
+	// sort the ids and check for identical neighbors
+	sort.Strings(ids)
```
Comment (jtolds) [inline pkg/storage/ec/client.go]: this works. another way is to use a map

Comment (kaloyan-raev) [reply pkg/storage/ec/client.go]: True. I just decided to go with the solution that is easiest to read. I was pretty disappointed there isn't any function for this in the standard Go packages.

Comment (jtolds) [reply pkg/storage/ec/client.go]: i agree it's a bummer the stdlib doesn't have something more nice for this, but the following is a bit shorter:

```
  seen := map[string]bool
  for _, n := range nodes {
    if seen[n.Id] {
      return false
    }
    seen[n.Id] = true
  }
  return true
```

Comment (kaloyan-raev) [reply pkg/storage/ec/client.go]: Yes, it looks better. Perhaps, we should have a utility package with such gems.

**Pull request merged**